transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -33,6 +33,34 @@
 logger = logging.getLogger(__name__)
 
 
+def create_naptan_flexible_zone_df_from_queryset(queryset):
```

Comment:
Add docstrings where applicable

Reply:
added the docstring to newly added functions

Reply:
added